lib/rmt/config.rb

```diff
@@ -38,6 +37,10 @@ def mirror_src_files?
       ActiveModel::Type::Boolean.new.cast(Settings.try(:mirroring).try(:mirror_src))
     end
 
+    def mirror_drpm_files?
+      ActiveModel::Type::Boolean.new.cast(Settings.try(:mirroring).try(:mirror_drpm)) || false
```

Comment:
```suggestion
      !!Settings.try(:mirroring).try(:mirror_drpm)
```

Reply:
The `!!` "method" won't convert a string like `"'false'"` to `false`, but rather to `true`. I considered this scenario in case the user surrounds a boolean with quotes, making it a string in the YAML file.

There is an example in the `RMT::Config` [spec, line 89](https://github.com/SUSE/rmt/pull/1308/files#diff-9e0b1824898f095a1821711cea640a8250ed44318d34f15fd3d452adab3f6eb1R89-R93).